src/components/billingRecord/IFXBillingRecordList.vue

```diff
@@ -495,9 +499,10 @@ export default {
     },
     addNewTransaction(item) {
       const orgBillingRec = item.orgRec
-      const { charge, rate, description, author } = item
+      const { charge, decimalCharge, rate, description, author } = item
```

Comment:
Is `decimalCharge` ever likely to be in the same transaction as `charge`?

Reply:
No. I'll take that out